src/containers/NamespacesDropdown/NamespacesDropdown.test.js

```diff
@@ -41,8 +41,9 @@ it('NamespacesDropdown renders items', () => {
   namespaces.forEach(item => {
     expect(queryByText(new RegExp(item, 'i'))).toBeTruthy();
   });
-  getAllByText(/namespace-/i).forEach(node => {
-    expect(namespaces.includes(getNodeText(node))).toBeTruthy();
+  const sortedNamespaces = namespaces.sort();
```

Comment:
This is an in-place sort so `namespaces` will be re-ordered here too

Reply:
Indeed, good catch.